bin.src/EventListener.py

```diff
@@ -29,6 +29,7 @@
 #                 they are happening for debugging.
 #
 #
+from __future__ import print_function
 import lsst.ctrl.events as events
 import lsst.pex.logging as logging
```

Comment:
You switched the examples away from `pex_logging` but we still have it in the code itself?